dll/win32/setupapi/query.c

```diff
@@ -20,6 +20,27 @@
 
 #include "setupapi_private.h"
 
+#ifdef __WINESRC__
```

Comment:
Why the WINESRC name?
am i just silly i dont remember us ever using this?>

Reply:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=_WINESRC_